### PR TITLE
Fix deprecation warnings in django 1.8 fixes #60

### LINKED ISSUE #60: Deprecated importlib module and deprecated get_cache
Django's importlib module will be removed with the next version (1.9).
Also, 'get_cache' is deprecated in favor of 'caches'.

Please replace in utils.py:

> from django.utils.importlib import import_module

With the Python importlib:

> from importlib import import_module

You can modify the get_cache import like that, so it works with all Django versions:

> try:
> ----from django.core.cache import caches
> except:
> ----from django.core.cache import get_cache as caches

Then load the cache like so:

> cache = hasattr(caches, '__call__') and caches(cache_name) or caches[cache_name]

Wonderful tool by the way! Thank you :-)

### PR DESCRIPTION
**Pull request by jieter**
The fix.

### CODE REVIEW

**Comment by jieter** [comment]
Hmm, CI fails, I'll setup a test env and at least fix the test for that environment.

**Comment by jieter** [comment]
almost there, only have to fix importlib for python 2.6. I guess we have to fall back to django's importlib for that...

File: ratelimit/utils.py [hunk 1]
```diff
@@ -1,12 +1,16 @@
 import hashlib
+from importlib import import_module
 import re
 import time
 import zlib
 
 from django.conf import settings
-from django.core.cache import get_cache
+try:
+    from django.core.cache import caches
```
Comment (jsocol) [inline ratelimit/utils.py]: I find it less awkward to do 

```python
try:
    from django.core.cache import caches
    def get_cache(name):
        return caches[name]
except ImportError:
    from django.core.cache import get_cache
```

Comment (jieter) [inline ratelimit/utils.py]: Agreed, less confusing. I'll update.

File: ratelimit/utils.py [hunk 2]
```diff
@@ -1,12 +1,16 @@
 import hashlib
+from importlib import import_module
```
Comment (jsocol) [inline ratelimit/utils.py]: I just did this here https://github.com/jsocol/django-jsonview/pull/22

**Comment by jieter** [comment]
@jsocol everything is green now at travis.

**Comment by jieter** [comment]
@jsocol should I squash the commits, or is this good to merge?

**Comment by jsocol** [comment]
@jieter would you mind squashing?

Sorry, I'm really swamped at work and it'll probably be next week before I really cycle back to this and other open source stuff

**Comment by jieter** [comment]
@jsocol never mind, no hurry.

Just pushed the squashed commit.

**Comment by jsocol** [comment]
Thanks! And thanks for understanding. I'll get back as soon as I can

**Comment by jieter** [comment]
@jsocol Can you maybe find some time to look into this?

**Comment by Sveder** [comment]
@jsocol Anything we can do to help you push this through to pypi?
That warning is making me twitch :)

**Comment by jieter** [comment]
Seems #70 fixes the things this PR was trying to fix.

@jsocol #60 can be closed too now.

**Comment by jieter** [comment]
@jsocol also: please push new version to PyPI

**Comment by jsocol** [comment]
Working on it right now

**Comment by jsocol** [comment]
v0.6.0 is up on PyPI

**Comment by Sveder** [comment]
Awesome, thank you!

**Comment by jieter** [comment]
thanks!

**Comment by adamchainz** [comment]
thx coveralls

**Pull request closed without merging**